src/resources.cpp

```diff
@@ -56,7 +57,10 @@ class FileResourceStream : public ResourceStream
 public:
     FileResourceStream(string filename)
     {
-        open_success = stream.open(filename);
+        if(!std::filesystem::is_regular_file(filename.c_str()))
```

Comment:
This can throw an filesystem_error, need to catch that or use the non throwing version.

Reply:
Didn't see this sorry, seems my eyes are not so used yet to noexcept :) error_code.message() *may be implemented* as throwing from this doc   but I think it may be overkill to try/catch it (and I don't believe it will either). But I can if you wish.

Reply:
And did not check when LOG initialisation is done in details neither :/

Reply:
Should be fine now, don't expect .message() to throw in any sane condition. I only noticed that it could throw because my eyes fell on the description part that said it. It is easy to miss, and the standard library isn't very consistent with exceptions.